Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorNodes.h

```diff
@@ -138,7 +138,7 @@ namespace AZ::DocumentPropertyEditor::Nodes
 
         static constexpr auto EnumType = TypeIdAttributeDefinition("EnumType");
         static constexpr auto EnumUnderlyingType = TypeIdAttributeDefinition("EnumUnderlyingType");
-        static constexpr auto EnumValue = AttributeDefinition<Dom::Value>("EnumValue");
+        static constexpr auto EnumValue = EnumValueAttributeDefinition("EnumValue");
```

Comment:
Sorry, refresh my memory on what `EnumValue` is used for? On a superficial search it looks like it only takes one enum value, so I just want to make sure I understand what the RPE-side handlers are trying to do with it before I dig in further on the DPE side.

Reply:
`EnumValue` is an internal attribute that the serialize context uses. I don't see it being used to reflect a given value anywhere, with a global search. I do see its case in `PropertyEnumBoxComboCtrl::ConsumeAttribute` (ln 107-ish) being hit often enough when loading DPEs for some assets/components. Some of those have broken combo boxes, hence the trying to fix `EnumValue` parsing.

I guess it's possible we don't actually need to handle this attribute at all in the reflection bridge since it's supposed to be internal to the serialize context? The legacy reflection bridge does currently attempt to parse them; it just fails to retrieve something that the `AttributeReader` can read.

Reply:
More comments below.